model-optimizer/extensions/ops/BatchNormInference.py

```diff
@@ -18,18 +18,23 @@
 from mo.ops.op import Op
 
 
-class BNOp(Op):
+class BatchNormInference(Op):
     """
     Empty Op for BN layer. It will be replaced by BNToScaleShift FrontReplacer
     """
-    op = 'BN'
-    enabled = True
+    op = 'batchNormInference'
+    enabled = False
 
     def __init__(self, graph: Graph, attrs: dict):
         super().__init__(graph, {
-            'type': None,
-            'op': __class__.op,
+            'type': self.op,
+            'op': self.op,
             'in_ports_count': 5,
             'out_ports_count': 1,
-            'infer': None
+            'infer': self.infer
         }, attrs)
+    @staticmethod
+    def infer(node):
+        node.out_port(0).data.set_shape(node.in_port(0).data.get_shape())
+        if node.in_port(0).data.get_value() is not None:
+            node.out_port(0).data.set_value(node.in_port(0).data.get_value())
```

Comment:
The value propagation is wrong. Please, remove it

Reply:
fixed